QKSMS/src/main/res/values-pl/strings.xml

```diff
@@ -359,44 +354,41 @@
-  <string name="mms_setup_one_configuration_found_title">One configuration found</string>
-  <string name="mms_setup_one_configuration_found_body">We have found an MMS configuration for the %s carrier. Does that sound right?</string>
-  <string name="mms_setup_multiple_configurations_found_title">Multiple configuration found</string>
-  <string name="mms_setup_multiple_configurations_found_body">We have found multiple MMS configurations. Choose your carrier from the list, or press N/A if your carrier isn\'t on the list.</string>
-  <string name="mms_setup_next_steps_title">Next steps</string>
-  <string name="mms_setup_next_steps_body">Visit the MMS section of Settings to try automatic configuration again, contact support, or enter your own configuration.</string>
-  <string name="mms_setup_success_title">Success!</string>
-  <string name="mms_setup_success_body">If you encounter any issues, visit the MMS section of Settings to try automatic configuration again, contact support, or enter your own configuration.</string>
-  <string name="na">N/A</string>
-  <string name="no">No</string>
-  <string name="pref_automatically_configure_mms">Automatically Configure MMS</string>
-  <string name="pref_mms_contact_support">Contact Support</string>
-  <string name="pref_category_sending_mms">Sending MMS</string>
-  <string name="pref_category_mms_configuration">MMS Configuration</string>
-  <string name="pref_mms_contact_support_summary">Contact the QKSMS team for help setting up MMS</string>
-  <string name="pref_markdown_enabled">Text formatting</string>
-  <string name="pref_markdown_enabled_summary">Use markdown syntax to get bold, underline, and italics in your messages</string>
-  <string name="want_read_indicators">Want read indicators?</string>
+  <string name="sending_blocked_welcome_screen_message">Bez pośpiechu! Już za chwilę będziesz mógł wysyłać SMS-y!</string>
+  <string name="notification_colored_icon_ready">Naciśnij, aby dodać skrót do ekranu głównego.</string>
+  <string name="notification_title_colored_icon_ready">Kolorowa ikona QKSMS jest już gotowa</string>
+  <string name="icon_ready">Twoja nowa ikona jest gotowa!</string>
+  <string name="icon_ready_message">Możesz ją teraz ponownie dodać do ekranu głównego z nowym kolorem.</string>
+  <string name="dont_ask_again">Nie pytaj ponownie</string>
+  <string name="mms_setup_set_up_mms_title">Skonfigurować MMS?</string>
+  <string name="mms_setup_set_up_mms_body">Wygląda na to, że nie masz skonfigurowanych MMS w QKSMS. Naciśnij OK, aby automatycznie skonfigurować MMS.</string>
+  <string name="mms_setup_setting_up_later_title">Skonfiguruj ustawienia MMS później.</string>
+  <string name="mms_setup_setting_up_later_body">Przejdź do sekcji MMS w Ustawieniach programy, aby spróbować ponownie automatycznej konfiguracji, skontaktuj się ze wsparciem lub wprowadź własne ustawienia.</string>
```

Comment:
This one is weird. I'd probably change it to:
'Przejdź do sekcji MMS w ustawieniach programu, aby spróbować automatycznej konfiguracji, skontaktować się ze wsparciem lub wprowadzić własne ustawienia.'